rd-net/RdFramework.Reflection/RdFramework.Reflection.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks>netstandard2.0;net35;net472</TargetFrameworks>
+        <TargetFramework>net8.0</TargetFramework>
```

Comment:
Before merging, we'll have to restore the compatibility with the older frameworks.